main.tf

```diff
@@ -324,10 +327,12 @@ resource "helm_release" "karpenter" {
     logLevel: info
     dnsPolicy: Default
     replicas: "${local.karpenter.replicas}"
-    resources:
-      requests:
-        cpu: "0.25"
-        memory: 256Mi
+    podAnnotations: ${jsonencode(local.karpenter.pod_annotations)}
```

Comment:
I guess the other jsonencode can be removed. 

Would need to confirm if mixing hcl and json syntax works together. I would assume it does